src/utils.ts

```diff
@@ -393,14 +393,24 @@ export async function querySubgraphWithRetry(
   if (json.errors) {
     console.error("querySubgraph:", json.errors);
 
-    if (retries > 0) querySubgraphWithRetry(url, query, variables, retries - 1);
-    else return null;
+    if (retries > 0) {
+      console.debug("Retrying query subgraph, retry countdown:", retries);
+      await querySubgraphWithRetry(url, query, variables, retries - 1);
```

Comment:
```suggestion
      return await querySubgraphWithRetry(url, query, variables, retries - 1);
```

Reply:
I think if you don't return here, then it would always cache `null` if the function goes into retry.

Reply:
Is fixed in [b40a9de](https://github.com/radicle-dev/radicle-interface/pull/407/commits/b40a9de845332d499c0e5d8814bf771ad786c448)